services/api/src/typeDefs.js

```diff
@@ -869,7 +869,7 @@ const typeDefs = gql`
     """
     Reference to EnvironmentStorage API Object, which shows the Storage consumption of this environment per day
     """
-    storages: [EnvironmentStorage]
+    storages(startDate: Date, endDate: Date, limit: Int, claim: String): [EnvironmentStorage]
```

Comment:
This doesn't do what I expect. The other places where we use `limit` as a field limits the number of records returned, but this is how many days in the past to return results. So maybe a name like `lastDays` or `daysAgo` is better.